src/test/java/org/fundacionjala/coding/danielM/EANValidatorTest.java

```diff
@@ -0,0 +1,90 @@
+package org.fundacionjala.coding.danielM;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Administrator on 3/3/2017.
+ */
+public class EANValidatorTest {
+    /**
+     * Test when the EAN string number has exactly 13 digits.
+     */
+    @Test
+    public void testCheckLengthWhenTheEANStringNumberHasExactly13Digits() {
+        // given:
+        final String eanStringNumber = "4003301018398";
+
+        // when:
+        final boolean actualResult = EANValidator.checkCorrectLength(eanStringNumber);
+
+        // then:
+        final boolean expectedResult = true;
+        assertEquals(expectedResult, actualResult);
```

Comment:
instead of using an assertEquals here
you can use an assertTrue or assertFalse
Please apply it to all your unit tests.